x-pack/legacy/plugins/apm/server/lib/settings/agent_configuration/get_service_names.ts

```diff
@@ -46,5 +46,6 @@ export async function getServiceNames({ setup }: { setup: Setup }) {
 
   const resp = await client.search(params);
   const buckets = idx(resp.aggregations, _ => _.services.buckets) || [];
-  return buckets.map(bucket => bucket.key).sort();
+  const serviceNames = buckets.map(bucket => bucket.key).sort();
+  return ['ALL_OPTION_VALUE', ...serviceNames];
```

Comment:
const?

Reply:
Good catch to both of these. I already have a constant but it was defined in the frontend and as a temp workaround I did this.